### ISSUE TITLE
[NEXT] `prefix` strategy throws `Error: nuxt instance unavailable`

### ISSUE THREAD

**Issue by cexbrayat** [nuxt-modules/i18n#1555]
### Version
@nuxtjs/i18n: 8.0.0-alpha.3
nuxt: 3.0.0-rc.11

### Nuxt configuration

Please change to `[x]` if relevant for this issue:

 - [x] Applies to a site deployed to a static server (site generated with `nuxt generate`)
 - [ ] Applies to a site deployed to a server with a Node backend

### @nuxtjs/i18n configuration

```ts
i18n: {
    locales: ['en', 'fr'],
    defaultLocale: 'en',
    strategy: 'prefix',
    vueI18n: {
      fallbackLocale: 'en',
      messages: {
        en: {
          welcome: 'Welcome',
        },
        fr: {
          welcome: 'Bienvenue',
        },
      },
    },
  }
```

### Reproduction Link

https://stackblitz.com/edit/github-ih5gur?file=nuxt.config.ts

### Steps to reproduce

```
npm run generate
```

### What is Expected?

Routes with `en` and `fr` should be generated without errors

### What is actually happening?

```
ℹ Prerendering 4 initial routes with crawler                       nitro 17:00:10
  ├─ /en (209ms)                                                   nitro 17:00:11
  ├─ /fr (4ms)                                                     nitro 17:00:11

 ERROR  [unhandledRejection] nuxt instance unavailable                   17:00:11
```

**Comment by kazupon** [comment]
Thank you for your reporting!

Maybe, This issue is an issue on nuxt.

See https://github.com/nuxt/nuxt.js/issues/15112

**Comment by danielroe** [comment]
@kazupon The issue is that Nuxt needs to do a transform for any async middleware to ensure that the Nuxt instance persists  across `await`.

Hopefully https://github.com/nuxt-community/i18n-module/pull/1558 will solve this issue 🙏

**Comment by kazupon** [comment]
Thanks! I've just merged your PR!

This issue will be released for next v8.0.0-alpha.4
If you would try to use, you can edge version.

Thanks!

**Comment by cexbrayat** [comment]
I confirm this is fixed in the edge version. Thanks @kazupon and @danielroe !
